tests/ops/test_simple_gla.py

```
@@ -329,6 +329,153 @@ def test_chunk_varlen(
     assert_close('dh0', ref_dh0, tri_dh0, 0.005)
 
 
+@pytest.mark.parametrize('B', test_b_list)
```

Ruff (0.11.9)

Undefined name `test_b_list`

(F821)